test/helpers/kubectl.go

```diff
@@ -1130,3 +1130,83 @@ func (kub *Kubectl) GetCiliumPodOnNode(namespace string, node string) (string, e
 
 	return res.Output().String(), nil
 }
+
+// CiliumPreFlightCheck validates that all prefligth are correct. If one of the
```

Comment:
prefligth --> preflight

Reply:
Specify that it checks that various subsystems within Cilium are in a good state.